Automate (#7)
byu-oit/developer-portal-definitions: README.md

# developer-portal-definitions

> This is where all of the API Documentation for the new developer portal at [developer.byu.edu](https://developer.byu.edu) lives.

Reviewer: Let's provide a deeper link. I also made the sentence more terse.
```suggestion
This is the source for API documentation found at [developer.byu.edu/api-search](https://developer.byu.edu/api-search).
```

## To add an API to the Developer Portal
- Clone this repo
- Make a new branch off of main
- Add your swagger document to the `swagger` folder
- Commit your changes
- Make a pull request that requests one of the following people to review
  - Rick Winscot
  - Gary Crye

>   - Brock May

Reviewer: Let's use a GitHub team instead of listing individuals. And better yet, let's use CODEOWNERS to automatically request reviews if that team's going to be gatekeeping the repo anyway.

Reviewer: The first half of my comment has been resolved.
